ESH-INF/thing/thing-types.xml

```diff
@@ -119,5 +120,11 @@
         <label>Play URI</label>
         <description>Play the stream at given uri</description>
     </channel-type>
+    
+    <channel-type id="playFile" advanced="true">
+        <item-type>String</item-type>
+        <label>Play file</label>
+        <description>Play a file, located on your Volumio2 device at the given absolute path, e.g. "mnt/INTERNAL/song.mp3"</description>
```

Comment:
There´s a little typo within the example for the absolute path... :) the path is not absoulte... I guess it should look like this...
```
/mnt/INTERNAL/song.mp3
```

Reply:
@thjunge11 Many thanks for you pull request... 👍

Reply:
@patrickse: The path needs to be absolute but without leading "/" for the uri in the json.

Volumio does find:  
{code}
2017-10-04T16:31:02.447Z - info: ADDING THIS ITEM TO QUEUE: {"service":"mpd","title":"","uri":"mnt/INTERNAL/sound.wav"}
{code}
but would not find:

{code}
2017-10-04T16:31:02.447Z - info: ADDING THIS ITEM TO QUEUE: {"service":"mpd","title":"","uri":"/mnt/INTERNAL/sound.wav"}
{code}

Reply:
Oh ok.. got it... so anyway thanks for the cool addition... ;)